docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
     environment:
       - ADMINMAIL=admin@kimai.local
       - ADMINPASS=changemeplease
-      - DATABASE_URL=mysql://kimaiuser:kimaipassword@sqldb/kimai
```

Comment:
`database` because it's a more generic term. And yes this needs also to be changed both in the container name and in the `DATABASE_URL` string (as my commit indicates, normally)

Reply:
Reverted to `sqldb`.